lib/utils.js

```diff
@@ -62,7 +62,7 @@ var installIfMissing = exports.installIfMissing = function(root, module, range)
         module = module + '@' + range;
       }
       console.log('Installing ' + module);
-      return runCommand('npm', ['install', module, '--loglevel', 'error']);
+      return runCommand('npm', ['install', module, '--no-shrinkwrap', '--loglevel', 'error']);
```

Comment:
There is no available documentation for `--no-package-lock` (checked both `man npm-install` and npm master), but `--no-shrinkwrap` is documented and does the same thing so I would rather rely on that.